spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md

```diff
@@ -205,3 +213,62 @@ This approach is depicted in the following Figure that shows an overview of the
   The unbonding operation is completed later once the provider CCV module registered maturity notifications for `VSC2` from all consumer chains.
 
 ![Completion of Unbonding Operations](./figures/ccv-unbonding-overview.png?raw=true)
+
+### Consumer Initiated Slashing
+[&uparrow; Back to Outline](#outline)
+
+For the [Security Model](#security-model) to be preserved, misbehaving validators MUST be slashed (and MAY be jailed, i.e., removed from the validator set). 
```

Comment:
The actual requirement is "misbehaving validators MUST be punished". This punishment can come in the form of slashing and maybe jailing. IMO, slashing is mandatory. Jailing without slashing will not be enough (since the attacker may leave the set anyway after an attack). Slashing without jailing will still satisfy the safety requirement, since it will act as a deterrent agains misbehavior. I agree though that both are better.